### PR TITLE
3. get /api/categories

### PR DESCRIPTION
**Pull request by Tucker-Ajai**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    ];
+
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.categories).toEqual(result);
+      });
+  });
+  test("When unable to access the server, Funtion responds with error", () => {
```
Comment (MKD01) [inline __tests__/app.test.js]: This is a very interesting test, but is not needed. Having a 500 error handler is absolutely fine but testing for this requires you to intentionally break the server/data which is not something we should expect to happen or need to test for.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("Get /api/categories", () => {
+  test("Function able to recieve a positive response", () => {
+    return request(app).get("/api/categories").expect(200);
```
Comment (MKD01) [inline __tests__/app.test.js]: This assertion has also been indirectly tested for in the below test as well, so you do not need to separate these two out.

**Review by MKD01 — APPROVED** [review]
Great stuff so far, but there are a few things you can take a look at

File: __tests__/app.test.js (hunk 1 above)
Comment (Tucker-Ajai) [reply __tests__/app.test.js]: I have removed this from the test file

File: __tests__/app.test.js (hunk 2 above)
Comment (Tucker-Ajai) [reply __tests__/app.test.js]: I have condensed this into one test

**Comment by Tucker-Ajai** [comment]
I am unable to create a new pull request after pulling from the main branch today. Please have a look at the commits tab for the latest update

**Review by Keb84 — APPROVED** [review]
Nicely done. Ready to merge 💃

**Pull request merged**